src/UKHO.S100PermitService.StubService/StubData/UserPermits/response-200-licenceId-2.json

```
@@ -0,0 +1,13 @@
+{
+  "LicenceId": "2",
+  "UserPermits": [
+    {
+      "Title": "IHO Test System",
+      "UPN": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    {
+      "Title": "OeM Test 1",
+      "UPN": "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+    }
+  ]
+}
```

Can we please change the data fields name to camelCase please. Please do it in all scenarios response.json files
```suggestion
{
  "licenceId": "2",
  "userPermits": [
    {
      "title": "IHO Test System",
      "upn": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
    },
    {
      "title": "OeM Test 1",
      "upn": "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
    }
  ]
}
```

---

Done